clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -192,6 +196,13 @@ RValue CIRGenFunction::emitBuiltinExpr(const GlobalDecl &gd, unsigned builtinID,
   }
   }
 
+  // If this is an alias for a lib function (e.g. __builtin_sin), emit
```

Comment:
We should have @andykaylor take a look at this, this was a good way to get a bunch of stuff passing by ignoring the above code for most of these, so I don't know how much this will harm us.  But overall it looks reasonable to me.

Reply:
Sure, I will wait for him to take a look too

Thanks

Reply:
This looks good. I guess this had to be moved down because otherwise we would have emitted the library call for `creal` before we got to the code that handles it. In this location, we can intercept any call for which we have specific handling as the handling is upstreamed, but any builtin that hasn't been implemented yet will still get lowered to the library call. That's what we want.